Makefile

```diff
@@ -450,6 +453,16 @@ pillar-%: $(GOBUILDER) | $(DIST)
 clean:
 	rm -rf $(DIST) images/out pkg-deps.mk
 
+$(DOCKERFILE_FROM_CHECKER): $(DOCKERFILE_FROM_CHECKER)
+	make -C $(DOCKERFILE_FROM_CHECKER_DIR)
+
+lint: $(DOCKERFILE_FROM_CHECKER)
+	@echo Checking Dockerfiles for inconsistencies
+	$(DOCKERFILE_FROM_CHECKER) ./ \
+		-i ./eve-tools/bpftrace-compiler/Dockerfile \
+		-i pkg/optee-os/Dockerfile \
```

Comment:
No way to avoid it?... Maybe let it fail now, and that way, force the owners to fix the inconsistency?...

Reply:
If you really really really want a red pipeline on every PR, then let's introduce that in a new PR and not with this one, so that everybody gets warned publicly.

Reply:
Then, please, at least drop a TODO/FIXME comment close to these lines so we remember why this workaround was introduced.

Reply:
@OhmSpectator , @christoph-zededa , please, see https://github.com/lf-edge/eve/pull/4334#discussion_r1797043925